pages/en/about/index.mdx

```diff
@@ -4,11 +4,14 @@ layout: about
 ---
 
 <div className="my-4 flex items-center justify-center">
-  <img
-    alt="Node.js mascot"
-    src="/static/images/node-mascot.svg"
-    className="w-[100px]"
-  />
+  {
```

Comment:
Why are there brackets here?

Reply:
MDX:
```mdx
<img
  alt="Node.js mascot"
  src="/static/images/node-mascot.svg"
  width={100}
  height={114.7}
/>
```

Rendered HTML:
```HTML
<img alt="Node.js mascot" src="/static/images/node-mascot.svg" width="100" height="114.7">
```

----

MDX With Bracket:
```mdx
{
  <img
    alt="Node.js mascot"
    src="/static/images/node-mascot.svg"
    width={100}
    height={114.7}
  />
}
```

Rendered HTML:
```HTML
<img alt="Node.js mascot" loading="lazy" width="100" height="114.7" decoding="async" data-nimg="1" style="color:transparent" srcset="/_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&amp;w=128&amp;q=75 1x, /_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&amp;w=256&amp;q=75 2x" src="/_next/image?url=%2Fstatic%2Fimages%2Fnode-mascot.svg&amp;w=256&amp;q=75">
```

As far as I understand, when we try to use it without adding curly brackets, it uses the default HTML element. A similar situation exists in the anchor element. It is not right to solve this within this card, I will create a new issue about this problem